adapter/i_o.py

```diff
@@ -144,6 +141,8 @@ def load(
 
         Parameters:
 
+            ts_format: str
+                timestamp format. i.e. 'short', 'long'
```

Comment:
I would add basic examples here to show what to expect with "long" vs "short". I doubt most users would go and look at `mark_time`

Reply:
I agree. The easy solution is to copy the same text as in the dedicated method. Also, it please provide the default.

Reply:
I updated the docstring with three examples.